packages/config/config/devices/0x0234/templates/logic_group_template.json

```
@@ -0,0 +1,550 @@
+				"value": 4
+			},
+			{
+				"label": "Follow internal dimmer",
+				"value": 5
+			},
+			{
+				"label": "Follow internal dimmer inverted",
+				"value": 6
+			},
+			{
+				"label": "On for 5 sec",
+				"value": 7
+			}
+		]
+	},
+	"binary_switch_set": {
+		"label": "Binary Switch Set support",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "LED",
+				"value": 0
+			},
+			{
+				"label": "Switch and LED",
+				"value": 1
+			},
+			{
+				"label": "Pushbutton activated",
+				"value": 2
+			}
+		]
+	},
+	"pushbutton_function_zdb5100": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 4,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Toggle",
+				"value": 0
+			},
+			{
+				"label": "Automatic turn off after the time has expired",
+				"value": 1
+			},
+			{
+				"label": "Automatic turn on after the time has expired",
+				"value": 2
+			},
+			{
+				"label": "Always turn off or dim down",
+				"value": 3
+			},
+			{
+				"label": "Always turn on or dim up",
+				"value": 4
+			}
+		]
+	},
+	"input_funtion_relay": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Function setup of input",
+		"maxValue": 3,
+		"defaultValue": 3,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Level-controlled input",
+				"value": 1
+			},
+			{
+				"label": "Falling-edge controlled input",
+				"value": 2
+			},
+			{
+				"label": "Rising-edge controlled input",
+				"value": 3
+			}
+		]
+	},
+	"output_function_relay": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Output function",
+		"options": [
+			{
+				"label": "Controlled via Z-Wave messages",
+				"value": 0
+			},
+			{
+				"label": "Controlled by input 1",
+				"value": 1
+			}
+		]
+	},
+	"status_led": {
+		"$import": "../../templates/master_template.json#led_indicator_four_options",
+		"label": "Status of LED",
+		"defaultValue": 1,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Enabled",
+				"value": 1
+			},
+			{
+				"label": "Flashes in 1 second interval",
+				"value": 2
+			},
+			{
+				"label": "Flashes in 1/2 second interval",
+				"value": 3
+			}
+		]
+	},
+	"meter_type": {
+		"label": "Meter Type of input",
+		"description": "Meter Type of input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 5,
+		"defaultValue": 1,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Input not in use",
+				"value": 0
+			},
+			{
+				"label": "Electricity meter",
+				"value": 1
+			},
+			{
+				"label": "Gas meter.",
```

This is an example of what I meant with "check the style guide":
```suggestion
				"label": "Gas meter",
```